Example/src/screens/Inputs.tsx

```diff
@@ -269,18 +199,6 @@ export const Inputs = () => {
                     />
                 </View>
             </ExampleSection>
-            <ExampleSection title="UITransferInput">
```

Comment:
Isn't it provide a conflict with PR from @SAnatoliiS ? 🤔

Reply:
Maybe :) Someone will have to resolve it)